src/components/NoTrajectoriesText/index.tsx

```diff
@@ -28,7 +28,7 @@ const NoTrajectoriesText = ({
     return (
         <div className={styles.container}>
             <h3>Ways to get started:</h3>
-            <p>
+            <div>
                 <ol>
```

Comment:
another similar change: this isn't an ordered list

Reply:
well actually i think the design does show these bullet points as numbered, i guess that's why we had an `ol`?

Reply:
oh, you're right, nm